types/src/test/scala/roc/types/decoders/NumericDecodersSpec.scala

```diff
@@ -14,36 +14,26 @@ final class NumericDecodersSpec extends Specification with ScalaCheck { def is =
   Short Decoder
     must return the correct Short when Text Decoding a valid Short String                 ${ShortDecoder().testValidTextDecoding}
     must throw a ElementDecodingFailure when TextDecoding an Invalid Short String         ${ShortDecoder().testInvalidTextDecoding}
-    must return the correct Short when Binary Decoding a valid Short Byte Array           ${ShortDecoder().testValidBinaryDecoding}
```

Comment:
Gotcha, I see them now. Can you also remove the actual unit tests themselves? They aren't being run, but if we aren't going to run them, I'm 👍 on code deletion.